BusinessLayer/Terminate_Program.xaml

```diff
@@ -0,0 +1,79 @@
+<Activity mc:Ignorable="sap sap2010" x:Class="Terminate_Program" this:Terminate_Program.in_ProcessName="msedge" xmlns="http://schemas.microsoft.com/netfx/2009/xaml/activities" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:mva="clr-namespace:Microsoft.VisualBasic.Activities;assembly=System.Activities" xmlns:sap="http://schemas.microsoft.com/netfx/2009/xaml/activities/presentation" xmlns:sap2010="http://schemas.microsoft.com/netfx/2010/xaml/activities/presentation" xmlns:scg="clr-namespace:System.Collections.Generic;assembly=mscorlib" xmlns:sco="clr-namespace:System.Collections.ObjectModel;assembly=mscorlib" xmlns:this="clr-namespace:" xmlns:ui="http://schemas.uipath.com/workflow/activities" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">

```

Comment:
Not really suitable for BusinessLayer either. In ReFramework, you have a special folder called Framework where you already have terminating programs implemented. For now, maybe move it to serviceLayer, as it makes the most sense there.